src/AbstractPhpdocToTypeDeclarationFixer.php

```diff
@@ -185,6 +185,10 @@ protected function getCommonTypeFromAnnotation(Annotation $annotation, bool $isR
             return null;
```

Comment:
Hey,
I would like to consider this change to be a feature, not bugfix.
ref https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/blob/master/feature-or-bug.rst#feature